src/transpose.jl

```diff
@@ -4,54 +4,51 @@ end
 struct AdjointMap{T, A<:LinearMap{T}} <: LinearMap{T}
     lmap::A
 end
-(::Type{TransposeMap})(lmap::LinearMap{T}) where {T} = TransposeMap{T, typeof(lmap)}(lmap)
-(::Type{AdjointMap})(lmap::LinearMap{T}) where {T} = AdjointMap{T, typeof(lmap)}(lmap)
+
+TransposeMap(lmap::LinearMap{T}) where {T} = TransposeMap{T, typeof(lmap)}(lmap)
+AdjointMap(lmap::LinearMap{T}) where {T}   = AdjointMap{T, typeof(lmap)}(lmap)
```

Comment:
I don't know if these methods are still necessary; they probably date back to from before triangular dispatch existed.